src/pal/prebuilt/inc/corerror.h

```diff
@@ -225,6 +225,7 @@
 #define CORPROF_E_FUNCTION_IS_COLLECTIBLE EMAKEHR(0x137e)
 #define CORPROF_E_CALLBACK6_REQUIRED EMAKEHR(0x1380)
 #define CORPROF_E_CALLBACK7_REQUIRED EMAKEHR(0x1382)
+#define CORPROF_E_ASSEMBLY_ALREADY_DEFINED EMAKEHR(0x1383)
```

Comment:
It has been a while since I last added an error, but I believe you want to edit corerror.xml which will auto-generate this file during the build. If that no longer works we'll need to investigate what happened.

Reply:
Right now it doesn't work automatically. There is bat file in src\pal\prebuilt\corerror, but nobody calls it and nobody builds genheaders.exe from src\inc\genheaders.cs.
When I've manually built it and executed against *.h and *.rc it generated same file, as I've updated manually, except it missed copyright statement at the beginning.